CompuCell3D/core/Demos/PluginDemos/FocalPointPlasticityLinks/Simulation/FocalPointPlasticityLinksSteppables.py

```diff
@@ -0,0 +1,93 @@
+from cc3d.core.PySteppables import *
+
+import random
+
+
+class FocalPointPlasticityLinksSteppable(SteppableBasePy):
+
+    def __init__(self, frequency=1):
+
+        SteppableBasePy.__init__(self, frequency)
+
+    def step(self, mcs):
+        """
+        type here the code that will run every frequency MCS
+        :param mcs: current Monte Carlo step
+        """
+
+        # Let's see what links are up to concerning cell 1
+        cell1 = self.fetch_cell_by_id(1)
+        if cell1 is None:
+            return
+
+        # Let's see what cell 1 is linked to
+        fpp_data_list = self.get_focal_point_plasticity_data_list(cell1)
+        for link1j in self.get_fpp_links_by_cell(cell1):
+            # Knowing the link and one linked cell, we can get the other linked cell
+            cellj = link1j.getOtherCell(cell1)
+            print(f'\nLinked cell id: {cellj.id}')
+
+            # If data-handling is wrong, then the fetched data for this link might have multiple entries
+            fppd = [fppd for fppd in fpp_data_list if fppd.neighborAddress.id == cellj.id]
+            assert len(fppd) == 1
+            fppd = fppd[0]
+
+            # Let's get link data via the old CC3D method and through new link-based methods
+            print(f'Confirming linked cell by id: {cellj.id}, {fppd.neighborAddress.id}')
+            print(f'lambda_distance: {link1j.getLambdaDistance()}, {fppd.lambdaDistance}')
+            print(f'target_distance: {link1j.getTargetDistance()}, {fppd.targetDistance}')
+            print(f'max_distance: {link1j.getMaxDistance()}, {fppd.maxDistance}')
+            print(f'max_num_junctions: {link1j.getMaxNumberOfJunctions()}, {fppd.maxNumberOfJunctions}')
+            print(f'activation_energy: {link1j.getActivationEnergy()}, {fppd.activationEnergy}')
+            print(f'neighbor_order: {link1j.getNeighborOrder()}, {fppd.neighborOrder}')
+            print(f'init_mcs: {link1j.getInitMCS()}, {fppd.initMCS}')
+
+            # Let's check the link's derived properties
+            print(f'length: {link1j.length}')
+            print(f'tension: {link1j.tension}')
+
+        # Now let's quickly get the list of all cells linked to cell 1
+        linked_list = self.get_fpp_linked_cells(cell1)
+        [print(f'Cell 1 has a link with cell {c.id}') for c in linked_list]
```

Comment:
why list comprehension with prints?

Reply:
Just demonstrating new functionality. No preference on how we show it off.